rook/Tester.py

```diff
@@ -176,15 +176,14 @@ def get_valid_params():
     params.add_param('gold_files', '', 'Gold filenames')
     return params
 
-  def __init__(self, name, params, test_dir):
+  def __init__(self, _name, params, test_dir):
     """
       Initializer for the class.
       @ In, name, string, name of class
```

Comment:
name --> _name

Reply:
BTW, is the _name still used?

Reply:
I think it was used in the original MOOSE based system, but there is a name in the new MOOSE based system, so I think we can leave it for now.

Reply:
I see, thanks, could you also update the docstring to reflect the changes.

Reply:
Docstring updated.